.github/ISSUE_TEMPLATE/release.md

```diff
@@ -19,3 +19,30 @@ assignees: ''
 - [ ] Create release on GitHub
 - [ ] Update public docs
 - [ ] Announce release
+
+## Update bundled Windows dependencies
```

Comment:
Please capitalize `bundled` and `dependencies`.

Reply:
Please count me in for Flo's next copy and paste in-house training 😅